app/server_test.go

```diff
@@ -22,40 +22,30 @@
 
 package app_test
 
-import (
-	"os"
-	"os/exec"
-	"syscall"
-	"testing"
-	"time"
-
-	"github.com/stretchr/testify/assert"
-)
-
-func TestServerInterruptRC(t *testing.T) {
-	// build and run the binary, not "go run" as it modifies the exit code
-	build := exec.Command("go", "build", "-o", "./test-temporal", "../cmd/temporal")
-	build.Env = append(os.Environ(), "CGO_ENABLED=0")
-	err := build.Run()
-	if err != nil {
-		t.Fatal(err)
-	}
-	defer os.Remove("./test-temporal")
-
-	cmd := exec.Command("./test-temporal", "server", "start-dev", "--headless")
-	err = cmd.Start()
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	// ensure the dev server process is killed, even if SIGTERM fails
-	defer cmd.Process.Signal(syscall.SIGKILL)
-
-	// Wait for the app to start
-	time.Sleep(time.Second * 2)
-
-	cmd.Process.Signal(syscall.SIGTERM)
-	_ = cmd.Wait()
-
-	assert.Equal(t, 0, cmd.ProcessState.ExitCode())
-}
+// func TestServerInterruptRC(t *testing.T) {
```

Comment:
Intentional?

Reply:
just pushed everything, cleaning things up and will rebase Namespace Update on top of this